src/components/Review.jsx

```diff
@@ -71,8 +72,7 @@ const Review = () => {
         })}
       </List>
     );
-
-  const previewRender = singleArticle ? (
+  const previewRender = singleArticle.id && !singleArticle.published ? (
```

Comment:
Will the singlearticle state be updated on publish to allow it to not be taken into account?

Reply:
Yes